app/tutorial_servo_control_can/src/main.cpp

```diff
@@ -0,0 +1,61 @@
+#include "mbed.h"
```

Comment:
This looks good, it should work. If you haven’t already, you should take a look at the arm_lower app to see an example of setting up hardware CAN filtering.